command/repo.go

```diff
@@ -127,6 +127,15 @@ func runClone(cloneURL string, args []string) (target string, err error) {
 func repoClone(cmd *cobra.Command, args []string) error {
 	cloneURL := args[0]
 	if !strings.Contains(cloneURL, ":") {
+		if !strings.Contains(cloneURL, "/") {
+			ctx := contextForCommand(cmd)
+			currentUsername, err := ctx.AuthLogin()
```

Comment:
We no longer provide `AuthLogin` from config: https://github.com/cli/cli/pull/976/commits/767521c05556746b4241ac34d58d5bc7b788f04d

Please request `CurrentLoginName()` in this case. It will result in an additional API request, which you will have to stub in tests for this case. Thank you!

Reply:
I fixed this [here](https://github.com/cli/cli/pull/861/commits/e89ddd745df873c5ea825ca227fdcd79b955b06d). Please take a look, thank you!